doc/source/examples/multiphysics/capillary-wave/capillary-wave.rst

```diff
@@ -0,0 +1,379 @@
+
+.. math::
+  \Delta t_\sigma = \frac{\Delta x}{\sqrt{2} c_\sigma} = \sqrt{\frac{\hat{\rho}}{2\pi\sigma}{{\Delta x}^3}}
+
+with the shortest unambiguously resolved capillary wave having a wavelength of :math:`\lambda_\sigma = 2 \Delta x`.
+
+Therefore, in order to get stable simulation results, :math:`\Delta t < \Delta t_\sigma` should be respected. In this example, different time-steps will be used to explore the stability limit of Lethe's current implementation.
+
+
+--------------
+Parameter File
+--------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+Below, the ``simulation control`` subsection for the case of :math:`\Delta t \approx 0.95\Delta t_\sigma \approx 0.95(3.9 \times 10^{-9})\, \text{s}` is shown. For other cases, the ``time step`` value will change and accordingly the ``output frequency`` will also.
+
+The time integration is handled by a 2nd-order backward differentiation scheme (bdf2) with a constant time-step of :math:`\Delta t=3.7 \times 10^{-9} \, \text{s}`. To assess the stability of the simulation results, the wave is simulated for :math:`t_{\text{end}} = 50\omega_\sigma \approx 4.5 \times 10^{-5} \, \text{s}`.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method           = bdf2
+      set time end         = 0.000045
+      set time step        = .0000000037
+      set output name      = capillary-wave-TSM-0_95
+      set output frequency = 243
+      set output path      = ./output-TSM-0_95/
+    end
+
+Multiphysics
+~~~~~~~~~~~~
+
+The ``multiphysics`` subsection is used to enable the VOF solver.
+
+.. code-block:: text
+
+    subsection multiphysics
+      set VOF  = true
+    end 
+
+Initial Conditions
+~~~~~~~~~~~~~~~~~~
+
+In the ``initial conditions``, we define the initial height of the wave, such that the interface (:math:`\phi = 0.5` isocurve) lies at the right height.
+
+.. code-block:: text
+
+    subsection initial conditions
+      set type = nodal
+      subsection uvwp
+        set Function expression = 0; 0; 0
+      end
+      subsection VOF
+        set Function expression = if (y<=1e-6*cos(2*3.14159/1e-4*x), min(0.5-(y-1e-6*cos(2*3.14159/1e-4*x))/1e-6,1), max(0.5-(y-1e-6*cos(2*3.14159/1e-4*x))/1e-6,0))
+        subsection projection step
+          set enable           = true
+          set diffusion factor = 1
+        end
+      end
+    end
+
+Mesh
+~~~~
+
+In the ``mesh`` subsection, we define a subdivided hyper rectangle with appropriate dimensions. The mesh is initially refined :math:`5` times to ensure adequate definition of the interface.
+
+.. code-block:: text
+
+    subsection mesh
+      set type               = dealii
+      set grid type          = subdivided_hyper_rectangle
+      set grid arguments     = 4, 12 : -5e-5, -1.5e-4 : 5e-5, 1.5e-4 : true
+      set initial refinement = 4
```

Comment:
```suggestion
      set initial refinement = 5
```

Reply:
To agree with the text above

Reply:
I also caught this one in my previous commit, but I guess you were already past this in your review.